File: FE/src/interfaces/detalleProducto.ts
import { Guid } from "guid-typescript";
import { Insumo } from "./insumo";
import { Producto } from "./producto";

export interface DetalleProducto{
    Id: string,
    ProductoId: Producto,
    fechaCreacion: Date,
    Insumo: Insumo,
    Cantidad: number,
    PrecioInsumo: number,
    Total: number
}

File: BE/API/Models/DetalleProducto.cs
namespace API.Models;

public class DetalleProducto
{
    public Guid Id { get; set; }
    public int Cantidad {get; set;}
    public DateTime? FechaCreacion {get; set;} 
    public double precioInsumo { get; set; }
    public double total {get; set;}
    public Guid InsumoId { get; set; }
    public Guid? ProductoId { get; set; }
}
